backend/server.go

```diff
@@ -130,6 +138,8 @@ var MissingParamInUrlError error = errors.New("Request Url was missing a param k
 var RoleNoteFoundError error = errors.New("This role does not exist")
 var InvalidJSON error = errors.New("Invalid Json")
 
+var emptyJson = map[string]interface{}{}
```

Comment:
In my next comment, i recommended removing the need for this. If you want to keep an emptyjson equivalent. We should replace this with a constructor, since this var is not immutable it'll be easy to accidently modify it.